seatunnel-examples/seatunnel-flink-connector-v2-example/src/main/java/org/apache/seatunnel/example/flink/v2/KafkaToConsoleExample.java

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
seatunnel-examples module is only used by developers. So test cases should not be submitted